docs/core/proto-docs.md

```diff
@@ -578,10 +578,15 @@
     - [Period](#cosmos.vesting.v1beta1.Period)
     - [PeriodicVestingAccount](#cosmos.vesting.v1beta1.PeriodicVestingAccount)
     - [PermanentLockedAccount](#cosmos.vesting.v1beta1.PermanentLockedAccount)
+    - [TrueVestingAccount](#cosmos.vesting.v1beta1.TrueVestingAccount)
   
 - [cosmos/vesting/v1beta1/tx.proto](#cosmos/vesting/v1beta1/tx.proto)
+    - [MsgClawback](#cosmos.vesting.v1beta1.MsgClawback)
+    - [MsgClawbackResponse](#cosmos.vesting.v1beta1.MsgClawbackResponse)
```

Comment:
So, for bikeshedding, I'd suggest something that clearly pairs the account type and clawback messages.  I think "true vesting" needs to go, though, since it seems to imply that this implementation is the best one. 
Some suggestions:

- `ClawbackVestingAccount`, `MsgClawbackVesting`, `MsgCreateClawbackVestingAccount`
- `RevocableVestingAccount`, `MsgRevokeVesting`, and `MsgCreateRevocableVestingAccount`

I should not be the final authority on this.  Please ensure Dean and Zaki make the decision as to the names.

Reply:
I'll check.